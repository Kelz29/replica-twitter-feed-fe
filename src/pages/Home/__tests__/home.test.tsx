import HomePage from "../index";

describe("Home Page Test", () => {
  it("renders successfully", () => {
    expect(HomePage).toMatchSnapshot();
  });
});
