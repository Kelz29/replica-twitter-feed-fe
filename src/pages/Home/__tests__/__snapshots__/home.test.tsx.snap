// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Test renders successfully 1`] = `[Function]`;
